db/seeds/admin_users.rb

```diff
@@ -0,0 +1,6 @@
+if Rails.configuration.x.admin_portal.password.present?
+  AdminUser.find_or_create_by(username: 'apply_maintenance') do |admin_user|
+    admin_user.password = Rails.configuration.x.admin_portal.password
+    admin_user.email = 'apply-for-legal-aid@digital.justice.gov.uk'.freeze
```

Comment:
Should we define `'apply_maintenance'` and `'apply-for-legal-aid@digital.justice.gov.uk'` as constants in the `AdminUser` model?

Reply:
I don't think so - as I'm not sure how persistent they need to be. At the moment, I think they are just values for an initial instance of `AdminUser` so relevant to seeds, but quite possibly no relevance to the model going forward.

I could be wrong.

Reply:
fair enough!